cas-server-webapp/src/main/resources/messages.properties

```diff
@@ -181,11 +181,23 @@ screen.institution.login.heading=Sign in through institution
 screen.institution.login.select=Select your institution
 screen.institution.login.select.auto=Your institution
 screen.institution.login.select.all=Not your institution?
+screen.institution.login.select.unsupported=I can't find my institution
 screen.institution.login.message=If your institution has partnered with OSF, please select its name below and sign in with your institutional credentials.<br/><br/>If you do not currently have an OSF account, this will create one for you.
 screen.institution.login.osf=Log in with OSF
 screen.institution.login.consent.checkbox=I have read and agree to the <a style="white-space: nowrap" href=https://github.com/CenterForOpenScience/centerforopenscience.org/blob/master/TERMS_OF_USE.md>Terms of Use</a> and <a style="white-space: nowrap" href=https://github.com/CenterForOpenScience/centerforopenscience.org/blob/master/PRIVACY_POLICY.md>Privacy Policy</a>.
 screen.institution.login.consent.error.message=You must read and agree to the <span style="white-space: nowrap">Terms of Use</span> and <span style="white-space: nowrap">Privacy Policy</span>.
 screen.institution.login.select.error.message=You must select an institution.
+screen.institution.login.unknown=I can't find my institution
+
+# Unsupported Institution
+screen.unsupportedinstitution.login.message=Signing into the OSF with institutional credentials is enabled for <a href="{0}">OSF Institutions members</a>.  If your institution is not yet an OSFI member, choose one of the following sign-in methods.
+screen.unsupportedinstitution.login.existing.heading=I already have an OSF account
+screen.unsupportedinstitution.login.existing.osf.message=If you have already have an OSF password, you can sign in to the OSF
+screen.unsupportedinstitution.login.existing.osf.button=Log in with OSF
+screen.unsupportedinstitution.login.existing.institution.message=If you have not yet set an OSF password for your account, create a password
+screen.unsupportedinstitution.login.existing.institution.button=Set a password
+screen.unsupportedinstitution.login.new.heading=I want to register for an OSF account
+screen.unsupportedinstitution.login.new.message=<a href="{0}">Register for an OSF account</a> using an email address or ORCID iD
```

Comment:
Since it's oldCAS, I decided not to do too much "refactor", so let's keep these long var names.